io.openems.edge.io.shelly/src/io/openems/edge/io/shelly/shellyplus3em/Config.java

```diff
@@ -0,0 +1,29 @@
+package io.openems.edge.io.shelly.shellyplus3em;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+import io.openems.edge.meter.api.MeterType;
+
+@ObjectClassDefinition(//
+		name = "IO Shelly Plus3EM", //
```

Comment:
Official name seems to be `Shelly 3EM`. I am renaming everything (classes, package, etc.)